rustfmt.toml

```diff
@@ -1,3 +1 @@
 edition = "2021"
-wrap_comments = true
-imports_granularity = "Crate"
```

Comment:
I noticed since the change to defaulting to stable that these options don't work locally and spam the terminal unless one adds the `+nightly` arg and
```shell
Warning: can't set `wrap_comments = true`, unstable features are only available in nightly channel.
Warning: can't set `imports_granularity = Crate`, unstable features are only available in nightly channel.
Warning: can't set `wrap_comments = true`, unstable features are only available in nightly channel.
Warning: can't set `imports_granularity = Crate`, unstable features are only available in nightly channel.
Warning: can't set `wrap_comments = true`, unstable features are only available in nightly channel.
Warning: can't set `imports_granularity = Crate`, unstable features are only available in nightly channel.
...
Warning: can't set `wrap_comments = true`, unstable features are only available in nightly channel.
Warning: can't set `imports_granularity = Crate`, unstable features are only available in nightly channel.
```

Reply:
Blech. OK.

https://github.com/rust-lang/rustfmt/issues/4991

Reply:
thanks for creating the issues, I should have done that instead of hoping one of us would remember to add it back...